app/controllers/supervisor/base_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Supervisor::BaseController < ApplicationController
```

Comment:
Ideally this controller should be tested, especially the `must_be_supervisor` method.